vms/proposervm/vm.go

```diff
@@ -333,37 +333,96 @@ func (vm *VM) SetPreference(ctx context.Context, preferred ids.ID) error {
 		return err
 	}
 
-	// reset scheduler
-	minDelay, err := vm.Windower.Delay(ctx, blk.Height()+1, pChainHeight, vm.ctx.NodeID, proposer.MaxBuildWindows)
+	var (
+		childBlockHeight = blk.Height() + 1
+		parentTimestamp  = blk.Timestamp()
+		nextStartTime    time.Time
+	)
+	if vm.IsDurangoActivated(parentTimestamp) {
+		currentTime := vm.Clock.Time().Truncate(time.Second)
+		nextStartTime, err = vm.getPostDurangoSlotTime(
```

Comment:
nit: why do we call this `getPostDurangoSlotTime` instead of just `getDurangoSlotTime`?

Reply:
This was done to align with `PreDurango` and `PostDurango`... After `Durango` activates - we'll remove all the transition logic anyways.